services/core-web/src/components/mine/DigitalPermitCredential/ViewDigitalPermitCredential.tsx

```diff
@@ -0,0 +1,344 @@
+import "@ant-design/compatible/assets/index.css";
```

Comment:
does it break without the antd compatible css file? It's on my 🪓 list.

Reply:
Oh, I think I can remove it.  I started with the ESUP form for a template, so that's an artifact.  Will try without.